public/static/data/ocean-watch.json

```diff
@@ -858,25 +859,25 @@
                           "id": "3966fed9-ef22-4c2b-b082-839de71f7eb4"
                         },
                         {
-                          "query": "https://wri-rw.carto.com/api/v2/sql?q=WITH r as (SELECT ROW_NUMBER() OVER (ORDER BY total DESC) AS ranking, rw_country_code as country_code, rw_country_name as country_name, total/100. as x, year FROM ene_012_electricity_access_edit WHERE year = 2018 AND year IS NOT NULL AND total IS NOT NULL ORDER BY total DESC), v as (SELECT count(distinct(country_code)), 'BRA' as country_code  FROM ene_012_electricity_access_edit) SELECT * FROM r INNER JOIN v ON r.country_code = v.country_code WHERE r.country_code = 'BRA'",
-                          "text": "Some highlight text about why this matters",
-                          "format": ".2f",
-                          "unit": "%"
+                          "query": "https://wri-rw.carto.com/api/v2/sql?q=SELECT 0.123456 AS value",
+                          "text": "Test text about why this value is relevant",
+                          "format": ".4p",
+                          "unit": ""
                         }
                       ]
                     },
                     {
-                      "id": "fisheries-subsidies",
-                      "title": "Fisheries subsidies (2009-19)",
+                      "id": "OECD-FAO-agricultural_outlook",
+                      "title": "OECD/FAO Agricultural Outlook 2020-2029",
                       "query": "https://wri-rw.carto.com/api/v2/sql?q=WITH r as (SELECT ROW_NUMBER() OVER (ORDER BY total DESC) AS ranking, rw_country_code as country_code, rw_country_name as country_name, total/100. as x, year FROM ene_012_electricity_access_edit WHERE year = 2018 AND year IS NOT NULL AND total IS NOT NULL ORDER BY total DESC), v as (SELECT count(distinct(country_code)), 'BRA' as country_code  FROM ene_012_electricity_access_edit) SELECT * FROM r INNER JOIN v ON r.country_code = v.country_code WHERE r.country_code = 'BRA'",
                       "format": ".2f",
                       "unit": "%",
                       "widgets": [
                         {
-                          "id": "eedaa69b-7d14-4541-9a0c-1033bcddd072"
+                          "id": "3966fed9-ef22-4c2b-b082-839de71f7eb4"
```

Comment:
@pkerins last Friday we performed a migration of the API to use the production API in the staging site. This means for you ids pointing to the staging API won't work anymore, would you mind updating this id with its equivalent (or any other) in production API? Thanks